containers/dev/docker-compose.yml

```diff
@@ -3,12 +3,13 @@ redis-dev:
   ports:
     - "6379:6379"
 elasticsearch-dev:
-  image: elasticsearch:1.7
+  image: elasticsearch:2.3.3
+  command: "elasticsearch -Des.cluster.name=elasticsearch -Des.node.master=true"
   ports:
     - "9200:9200"
     - "9300:9300"
 kibana-dev:
-  image: kibana:4.1
+  image: kibana:latest
```

Comment:
I'd prefer we figure out what the latest stable version is and explicitly use that, so we don't have a surprise kibana upgrade someday.